spec/features/guider_views_appointments_spec.rb

```diff
@@ -42,6 +42,78 @@
     end
   end
 
+  context 'Appointment is updated by someone else', js: true do
+    scenario 'viewing their appointments' do
+      @guider = create(:guider)
```

Comment:
:heavy_check_mark: couldn't think of a good name so went with yours. Seriously wondered about using `given_a_guider_with_appointments_and_another_without_any`